_layouts/home.html

```diff
@@ -12,14 +12,52 @@ <h2 id="countdown">&nbsp;</h2>
       This campaign is... something!
     </p>
   </div>
+  
+              <div class="carousel-inner" role="listbox">
 
-  <div class="flex row-xs-center ">
-    <div class="btn-group-lg">
-      <a class="btn btn-lg btn-default btn-next btn-xs-full" href="{{ '/host/' | prepend: site.baseurl }}">Become a host&nbsp;&nbsp;<i class="fa fa-play" aria-hidden="true"></i></a>
-      <a class="btn btn-lg btn-default btn-next btn-xs-full" href="{{ '/participant/' | prepend: site.baseurl }}">Attend an event&nbsp;&nbsp;<i class="fa fa-play" aria-hidden="true"></i></a>
-    </div>
-  </div>
+                  <div class="item active">
+
+                    <img src="../img/homepage_slide#1.jpg" alt="Bootstrap Touch Slider"  class="slide-image"/>
+
+                      <div class="container">
+                          <div class="row">
+                              <!-- Slide Text Layer -->
+                              <div class="slide-text slide_style_left">
+                                <h1 data-animation="animated zoomInRight">Looking forward to meet you in our conference!!!</h1>
+                                <p data-animation="animated fadeInLeft">The best is yet to come!!!</p>
+                                <a href="{{ '/participant/' | prepend: site.baseurl }}” target="_blank" class="btn btn-default" data-animation="animated fadeInUp">Become an host</a>
+                                <a href="{{ '/host/' | prepend: site.baseurl }}" target="_blank"  class="btn btn-primary" data-animation="animated fadeInDown">Attend an event</a>                              
+                                </div>
+                          </div>
+                      </div>
+                  </div>
+
+                  <div class="item">
+
+                      <!-- Slide Background -->
+                      <img src="../img/homepage_slide#2.jpg" alt="Bootstrap Touch Slider"  class="slide-image"/>
```

Comment:
please remove the # in the filename for all 3 images, otherwise the browser won't be able to load it